maven-java/kalix-spring-boot-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -282,7 +282,7 @@
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-test</artifactId>
-      <version>2.7.0</version>
+      <version>3.0.1</version>
```

Comment:
I suppose there's a reason for not using 3.0.2 here as well?

Reply:
Should be 3.0.2

Reply:
Actually, I had already noticed that 3.0.2 was out and was planning to update it. I will do it now.